### ISSUE TITLE
Abut the performance

### ISSUE THREAD

**Issue by SunQilin** [zhaoweicai/mscnn#24]
Hi @zhaoweicai,
   
I migrated your code to window platform(based on the happynear/caffe-windows), and ran your code as you describe in the README file. I trained the network of mscnn-7s-384-2x and mscnn-7s-576-2x. 
 While the performance of mscnn-7s-384-2x in you paper is 90.55 87.93 71.90(Easy, Moderate, Hard), I get 86.52 87.75 75.96(Easy, Moderate, Hard). 
 While the performance of mscnn-7s-576-2x in you paper is 94.08 89.12 75.54(Easy, Moderate, Hard), I get 86.96 88.33 77.96(Easy, Moderate, Hard).

I tried three time for mscnn-7s-384-2x, the results were close. The easy part was worse obviously and hard part was better. Are the results all right?

**Comment by zhaoweicai** [comment]
Hi @SunQilin 
It is good to see you have reasonable results. After the paper, I fixed some bugs and made some changes. In general, you should have better results with the code than Table 3 in the paper. A strange thing is your Easy performance is even worse than Moderate. Probably you did something wrong. I used KITTI devkit to evaluate. Make sure you use the same evaluation metric (code).

**Comment by SunQilin** [comment]
Hi, @zhaoweicai 
Sorry for replying you so late. I have been checking my code and running the code more times. I'm sure I used KITTI devkit to evaluate. The main difference was I used cudnn v5. Then I changed cudnn to v3 , but the performance was not improved yet.
I have some questions about the configure file of the network. Does "h576-2x" in your paper means input of height 576 without context encoding or dimensionality reduction? However, the network configure in the file "mscnn-7s-576-2x" of your your repository means input of height 576, with context encoding and dimensionality reduction? I trained the network as configure files in mscnn-7s-576-2x, and got a model which is 296M, similar as "h576-ctx-c" in your paper.

**Comment by zhaoweicai** [comment]
Hi @SunQilin 
Yes, you are right. "h576-2x" doesn't use "ctx-c". I don't provide every model in my ablation study. The model names in the code are a little bit inconsistent with the paper. But you always can check the "deploy.prototxt" to find out what it is. 
I think your results are reasonable for moderate and hard. But it is strange that easy is even worse than moderate. No body  else complains this problem, and it never happens to me. I will suggest you solve this problem first.

**Comment by lixincn2015** [comment]
HI, @SunQilin 
I wonder have you reproduced the performance of "7s-h576-2x" on validation set?
I have got some problem on reproducing the performance.
